internal/pkg/manifest/manifest.go

```diff
@@ -320,8 +320,10 @@ func (m *Manifest) DeleteRecordByKey(key model.Key) {
 }
 
 func (m *Manifest) GetParent(record model.Record) (model.Record, error) {
-	parentKey := record.ParentKey()
-	if parentKey == nil {
+	parentKey, err := record.ParentKey()
```

Comment:
Pridavam `error` aj do `ParentKey`, .. pouzije sa to pri implementaci `variables`, ak by bola nejaka chyba v referencii medzi `variables config <-> config`.